core/base/document.php

```diff
@@ -1065,6 +1066,29 @@ public function delete() {
 		return $deleted && ! is_wp_error( $deleted );
 	}
 
+	/**
+	 * On import Kit.
+	 *
+	 * Replace exported post-ids from settings with the new created posts ids.
+	 *
+	 * @since 3.6.0
+	 *
+	 * @param array $elements
+	 *
+	 * @param array $map_old_new_post_ids
+	 */
+	public function on_import_replace_dynamics_elements_id( &$elements, $map_old_new_post_ids ) {
+		foreach ( $elements as &$element ) {
+			$element = Template::on_import_replace_template_post_id( $element, $map_old_new_post_ids );
```

Comment:
Create an instance of the element and call its `on_import_replace_dynamic_content`
that will replace it's own settings and also call to `parent->on_import_replace_dynamic_content` for each dynamic tag in it.

The `on_import_replace_dynamic_content` should be in the `Controls_Stack` class, and do nothing by default.

For an element, it should run over all tags and call the `tag->on_import_replace_dynamic_content`